pkgs/development/python-modules/bokeh/default.nix

```diff
@@ -75,6 +75,7 @@ buildPythonPackage rec {
   '';
 
   meta = {
+    broken = true;
```

Comment:
please remove this. As @orivej pointed out, it builds, and if it doesn't, it needs to be fixed outside this PR.